### ISSUE TITLE
How to control the number of rows in equiv reasoning and notes

### ISSUE THREAD

**Issue by sangwinc** [maths/moodle-qtype_stack#864]
A user recently contacted me to ask the following:

Enclosed you find a screenshot with four input fields in one line. The first and third are of equivalence reasoning type,
the second and fourth are of notes type.

As you can see, the second (and third) contain explicit information about the steps that students perform on the equation. This is a form which is standard in German schools.

A small problem arises:  The input fields of eqreas and notes are of different height. How to define the height in advance so that both are automatically displayed with same height?

![screenshot](https://user-images.githubusercontent.com/781615/194916933-5bb32699-c359-439a-9130-c1ddc591538b.png)

**Comment by sangwinc** [comment]
Thanks for raising this.  There is currently no way for a user to control the number of rows in the inputs.  These are automatically resized on the number of rows here:

https://github.com/maths/moodle-qtype_stack/blob/master/stack/input/notes/notes.class.php#L55
https://github.com/maths/moodle-qtype_stack/blob/master/stack/input/equiv/equiv.class.php#L99

I can't remember why I chose 3+ and 5+ for equiv input and notes respectively, probably because didn't have a good reason not to!

Really, I think what you want is a compound input type which has both the equivalence reasoning and notes bound together so they always have the same number of rows.  Not easy to do in the current set up, but I've added it here as a request.
